addons/website/static/tests/tours/media_iframe_video.js

```diff
@@ -58,3 +58,63 @@ wTourUtils.registerWebsitePreviewTour(
         },
     ]
 );
+
+wTourUtils.registerWebsitePreviewTour(
+    "website_media_iframe_video_empty_url",
+    {
+        test: true,
+        url: "/",
+        edition: true,
+    },
+    () => [
+        wTourUtils.dragNDrop({
+            id: "s_text_image",
+            name: "Text - Image",
+        }),
+        {
+            content: "Select the image",
+            trigger: "iframe #wrap .s_text_image img",
+        },
+        {
+            content: "Click on replace media",
+            trigger: "[data-replace-media='true']",
+        },
+        {
+            content: "Click on video button",
+            trigger: "a:contains('Videos')",
+        },
+        {
+            content: "Enter text in video link input to enable add button",
+            trigger: "#o_video_text",
+            run: "text Test",
+        },
+        {
+            content: "Remove video link",
+            trigger: "#o_video_text",
+            run() {
+                const inputEl = this.$anchor[0];
+                inputEl.value = "";
+                inputEl.dispatchEvent(new Event("input", { bubbles: true }));
+            },
+        },
+        {
+            content: "Try clicking on 'Add' button",
+            trigger: ".modal-footer button:contains('Add')",
+        },
+        {
+            content: "Check that modal is still open with error notification",
+            trigger: ".o_select_media_dialog",
+            run() {
+                const notification = document.querySelector(
+                    ".o_notification_manager .o_notification_content"
+                );
+                if (
+                    !notification ||
+                    !notification.innerText.includes("The provided url is not valid")
+                ) {
+                    console.error("Expected error toast not found when adding empty video URL.");
+                }
+            },
+        },
```

Comment:
I believe this whole test could be the continuation test of [this](https://github.com/odoo/odoo/pull/234085/files#diff-9a84fb4cf938a561360e78056601c630e17b93544024e3927da4f1049c440139R36)
This way we have 2 things tested in one test.